### DISCUSSION TITLE
Changes in the date of receipt

### DISCUSSION THREAD

**Discussion by Fashak43** [gibbed/SteamAchievementManager#440] [General]
Is there any way to add a modified achievement date (I use the services of an interpreter because I don't speak English Well)

**Comment by hyacynthesizer** [comment]
Seconded, I'd also be interested in the ability to edit the unlock date, if possible (I understand that the program requires Steam to be online, so that eliminates the possibility of changing my computer's clock to be at the time I want, haha

**Reply by Fashak43** [reply]
I think it can be done. But you can check if you change the date (I have a 2007 account, and the account is registered in 2016)

**Reply by Fashak43** [reply]
Sorry, the translator is bad, I have an achievement of 2007

**Comment by hyacynthesizer** [comment]
Seconded, I'd also be interested in the ability to edit the unlock date, if possible (I understand that the program requires Steam to be online, so that eliminates the possibility of changing my computer's clock to be at the time I want, haha

**Comment by ultrakillinginvr** [comment]
###### i guess no
